content/article/the-perl-toolchain-summit-2025-needs-you.md

```diff
@@ -0,0 +1,135 @@
+contributor that is going to give their time and expertise for four
+days, to move the Perl toolchain forward. They give their time
+(sometimes having to take days off work, which is already a loss of
+income or holidays for them).
+
+This is why, since 2011, we've done our best to *at least partially
+refund* their travel and accommodation expenses when needed. Everyone
+who's attending the PTS should really *only* have to give four days of
+their life for it.
+
+If the PTS can't support its participants, then more and more of them
+are going have to either decline our invitation, or spend their own
+money, in addition to their time, to continue supporting the Perl
+Toolchain.
+
+**This is bad for Perl and CPAN.**
+
+Perl is not like Java or Go: it doesn't have a large corporation behind
+it. It's entirely supported by the community and its corporate sponsors.
+In others words, by you.
+
+## How much does a PTS cost, by the way?
+
+Let's do a quick back-of-the-enveloppe calculation, assuming:
+* hotel: 100€/night (most people are staying 5 nights, arriving the
+  day before and leaving the day after),
+* travel to Leipzig from Europe: 500€ round-trip,
+* travel to Leipzig from outside Europe: 1,500€ round-trip,
+* venue cost: 2,000€
+* lunch, snacks and coffee breaks: 15€/day/person
+
+We're expecting about 35 people coming (out of 44 invitations sent), 22
+from Europe, and 13 from outside Europe.
+
+That brings us to a grand total of 53,100 €, almost all costs
+considered, which I'll round up to **55,000€**. That's a lot of money.
+
+The organizers never actually spend that amount, because many of our
+attendees pay for themselves, or have their expenses covered by their
+employer (which we list as in-kind sponsors, alongside our financial
+sponsors).
+
+Our budget for 2025 is of 25,000 €: that is our financial sponsoring
+target, as well as the amount we expect to pay directly to various
+suppliers. The rest is covered by in-kind sponsors or the attendees
+themslves.
+
+## Our sponsors
+
+Here's our current list of confirmed sponsors for the Perl Toolchain
+Summit 2025. (We're currently in discussion with other sponsors, but
+nothing has been confirmed yet.)
+
+### Financial Sponsors
+
+These sponsors simply wire some money to [Les Mongueurs de
+Perl](https://www.mongueurs.net/), the French non-profit that
+handles the organization of the event (they get an invoice in
+return), and expect the organizers to spend it on PTS expenses
+(see above).
+
+Any money left over is used to kickstart the budget for the event the
+following year, as is our tradition since 2011.
+
+#### Gold Sponsors
+
+* [cPanel, Inc](https://cpanel.net/)
+
+#### Silver sponsors
+
+* [CosmoShop](https://www.cosmoshop.de/)
+* [Datensegler](https://datensegler.at/)
+* [SUSE](https://www.suse.com/)
+
+#### Bronze sponsors
+
+* [Simplelists Ltd](https://www.simplelists.com/),
+* [Ctrl O Ltd](https://www.ctrlo.com/),
```

Comment:
The trailing comma seems like a typo here, making the list style inconsistent with the majority of other list elements.